packages/teams-js/src/public/interfaces.ts

```diff
@@ -171,6 +171,11 @@ export enum FileOpenPreference {
   Web = 'web',
 }
 
+/**
+ * @deprecated
+ * As of 2.0.0-beta.1, please use {@link app.Context} instead.
+ * Represents structure of the received context message.
```

Comment:
Looks good, thanks for adding! nit: add a space between lines 176 and 177?

Reply:
Thanks. I'll make this change in the PR to increase descriptiveness to a Good standard and also incorporate the feedback on whether we use "." there or not.